server/controllers/previewCtrl.js

```diff
@@ -160,11 +160,15 @@ async function renderMemoPreviewPage(req, res, next) {
     applicationStore.setBrowserConfig(browser, serverPaths, apis, server.hostUrl)
     applicationStore.doSetLanguageIndex(userLang)
     const apiMemoData = await getMemoApiData('getDraftByEndPoint', { memoEndPoint })
+    const { semester, memoCommonLangAbbr } = apiMemoData
+    const memoLangAbbr = memoCommonLangAbbr || userLang
     const allApiMemoData = await getMemoApiData('getAllMemosByCourseCodeAndType', {
       courseCode,
       type: 'published',
     })
     applicationStore.memoDatas = allApiMemoData
+    // const ladokCourseRounds = await getCourseRoundsData(courseCode, memoLangAbbr)
+    // const ladokCourseData = await getLadokCourseData(courseCode, memoLangAbbr)
```

Comment:
remove?

Reply:
This has been fixed in the other PR :+1: